### ISSUE TITLE
Left/Right arrows shown first/last image

### ISSUE THREAD

**Issue by radpet** [IIIF/mirador#978]
When I load mirador with saved state if the state was showing first or last image the check for which arrow should not be shown is not working.

**Comment by rsinghal** [comment]
Yes, related to #300.  It only occurs when loading from saved state/settings.  We will look at it after we have incorporated `iiifManifestLayouts`
